packages/tools/client-debugger/client-debugger-view/src/components/RenderSumaryTree.tsx

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import { Tree, TreeItem, TreeItemLayout } from "@fluentui/react-components/unstable";
+
+/**
+ * Input to {@link RenderSumaryTree}
+ */
+type RenderSummaryTreeProps = React.PropsWithChildren<{
+	/**
+	 * Header label created by {@link RenderLabel}.
+	 */
+	header: React.ReactElement | string;
+
+	/**
+	 * List of child React Elements populated by recursion.
+	 */
+	children: React.ReactElement[];
```

Comment:
This property shouldn't be needed - the `children` prop is added via the `PropsWithChildren` type.

Reply:
Removed using  `React.Children`.